cmd/cli/kubectl-kyverno/variables/parse.go

```diff
@@ -0,0 +1,36 @@
+package variables
+
+import (
+	"strings"
+)
+
+func parse(vars ...string) map[string]string {
```

Comment:
I don't understand what this parse does, definitely need some tests to explain :)

Reply:
Will add unit tests